packages/field-highlighter/src/fields/vaadin-select-observer.js

```diff
@@ -9,32 +9,15 @@ export class SelectObserver extends FieldObserver {
   constructor(select) {
     super(select);
 
-    this.blurWhileOpened = false;
     this.overlay = select._overlayElement;
   }
 
-  addListeners(select) {
-    super.addListeners(select);
-
-    select.addEventListener('opened-changed', (event) => {
-      // When in phone mode, focus is lost when closing.
-      if (select._phone && event.detail.value === false) {
-        this.hideOutline(select);
-      }
-    });
-  }
-
   onFocusIn(event) {
     if (this.overlay.contains(event.relatedTarget)) {
       // Focus returns on item select, do nothing.
       return;
     }
 
-    if (!this.component._phone && this.overlay.hasAttribute('closing')) {
-      // Focus returns on outside click, do nothing.
```

Comment:
This logic is now covered by the block above with `event.relatedTarget` pointing to `vaadin-select-item` as the focus doesn't move to `<body>` on outside click anymore before it is restored.